src/tests/mod_test.ts

```diff
@@ -1,11 +1,11 @@
 import { assertStringIncludes } from "../dev_deps.ts";
-import hooksVersion from "../version.ts";
+import { VERSIONS } from "../libraries.ts";
 import { projectScripts } from "../mod.ts";
 
-Deno.test("projectScripts should return a check-update hook that points to its own version", () => {
+Deno.test("projectScripts should return a start hook that points to the enshrined deno-slack-runtime version", () => {
```

Comment:
Due to the change in `mod.ts` of using `import.meta.url`, I changed this test to inspect the contents of the `start` hook, which _will_ continue to enshrine a specific version tag for the deno-slack-runtime repo.